venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,776 @@
+                "aclweb.org/ACL/2022/Conference/Program_Chairs",
+                "aclweb.org/ACL/2022/Conference", 
+                "aclweb.org/ACL/2022/Conference/{sac_track}/Senior_Area_Chairs".format(sac_track = sac_name_dictionary[acl_submission.content['track']])
+                ],
+            nonreaders = [
+                "aclweb.org/ACL/2022/Conference/Paper{number}/Conflicts".format(number = acl_submission.number)
+                ],
+            writers = [
+                "aclweb.org/ACL/2022/Conference"
+                ],
+            signatures = [
+                "aclweb.org/ACL/2022/Conference"
+                ],
+            content = {
+                "authorids" : ["Anonymous"],
+                "authors":[f"aclweb.org/ACL/2022/Conference/Paper{acl_submission.number}/Authors"]
+            }
+        )
+        
+    blinded_note_posted = client.post_note(blinded_note)
+    print(blinded_note_posted.forum)
+    assert blinded_note_posted, print('failed to post blind note: ', blinded_note_posted.forum)
+    blind_submissions[blinded_note_posted.original] = blinded_note_posted
+    post_reviews(blinded_note_posted.forum, blinded_note_posted, arr_submission)
+
+# Post reviews (calls post_metareviews)
+def post_reviews(acl_blind_submission_forum, acl_blind_submission, arr_submission):
+    # Migrate all reviews from the original ARR Submission 
+    # Get invitation for that month's reviews 
+    conf_id = arr_submission.invitation.rsplit('/', 2)[0]
+    review_invitation_arr = '{conf_id}/Paper{number}/-/Official_Review'.format(conf_id = conf_id,number = arr_submission.number)
+    
+    # Get all reviews from the original ARR Submission 
+    arr_reviews = openreview.tools.iterget_notes(client, invitation = review_invitation_arr)
+    
+    # Iterate through each review and for each, create and post a new review 
+    for arr_review in arr_reviews:
+        print(arr_review.signatures)
+        if(arr_review.signatures[0] not in acl_reviews_dictionary):
+            print('arr review signatures', arr_review.signatures)
+            acl_review = openreview.Note(
+                forum = acl_blind_submission_forum,
+                replyto = acl_blind_submission_forum,
+                invitation = 'aclweb.org/ACL/2022/Conference/-/Official_Review',
+                signatures = arr_review.signatures,
+                readers = [
+                    'aclweb.org/ACL/2022/Conference/Program_Chairs'
+                ],
+                writers = [
+                    'aclweb.org/ACL/2022/Conference'
+                ],
+                content = arr_review.content
+            )
+            acl_review_posted = client.post_note(acl_review)
+            assert acl_review_posted, print('failed to post review ', acl_review.id)
+            acl_reviews_dictionary[acl_review_posted.signatures[0]] = acl_review_posted.replyto
+            post_metareviews(acl_blind_submission_forum, acl_blind_submission, arr_submission)
```

Comment:
post_metareviews() is being called in the for loop, so it is called every time a review is posted. This resulted in the metareview not being the top most reply in the forum

Reply:
I didn't realize that the metareview had to be the first one! I'll change this